src/containers/Inventory.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { connect } from 'react-redux'
+import { changeBlockName, toggleProperties } from '../actions'
+import Inventory from './../components/Inventory.js';
```

Comment:
I was just asking about the extra ./ in front of the ../ -- I'd never seen that syntax before, so I didn't know if this was a react thing or not.